alibi_detect/cd/mmd.py

```diff
@@ -92,6 +100,7 @@ def __init__(
         else:
             self._detector = MMDDriftTorch(*args, **kwargs)  # type: ignore
         self.meta = self._detector.meta
+        self._detector.backend = backend
```

Comment:
I assume this info is needed by the backend class so it can call it's own specific `get_config` method and save that info?

Reply:
I've got rid of this, as the `backend` is actually already stored in `self.meta['backend']`. I'm then accessing this in the base classes to reduce duplication a little.